cmd/pebble/cmd_boot.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"errors"
+	"os"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type cmdBoot struct {
+	clientMixin
+
+	Force bool `long:"force"`
+}
+
+var bootDescs = map[string]string{
+	"force": `Skip default checks`,
+}
+
+var shortBootHelp = `Bootstrap a system with Pebble running as PID 1`
+
+var longBootHelp = `
+The boot command checks for PID and working directory restrictions
+for Pebble and prepares required mountpoints before running the
+configured Pebble environment.
+`
+
+func (cmd *cmdBoot) Execute(args []string) error {
+	if len(args) > 1 {
```

Comment:
The command itself is a way to consume boot but we will have other means of consuming the feature. I think it would help avoid redundant code if we move all the checks to the internal pebble daemon.

Reply:
Oops! I think I made a typo here, and the expression would be `len(args) > 0`. I include this check as it's a common pattern I've seen in all other pebble commands.